tests/integration/test_charm.py

```diff
@@ -34,6 +34,7 @@
     verbs=None,
 )
 SELDON_CM_NAME = "seldon-config"
+DEFAULT_NAMESPACE = "default"
```

Comment:
Let's rename this to something like `WORKLOADS_NAMESPACE`. Mainly
1. In case we decide to use a different one in the future (i.e. create a profile?)
2. To make it more clear how this value will be used

Reply:
Updated!